travis/suites/suite-default

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
 bin/phpspec run -f dot
+bin/phpunit src/ -c phpunit.xml.dist
```

Comment:
as you've moved `phpunit.xml.dist` to the root, you should just be able to run `bin/phpunit` here.